test/binaryen.js/browser.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Binaryen</title>
+    <style type="text/css">
+
+* { margin:0; padding:0; font-family:inherit; }
+body {
+  margin:2em;
+  background:white;
+  color:black;
+  font:14px monospace;
+  white-space: pre;
+}
+
+    </style>
+    <script type="text/javascript" src="../../bin/binaryen.js"></script>
+    <!-- <script type="text/javascript" src="../../bin/binaryen-g.js"></script> -->
+  </head>
+  <body><script type="text/javascript">
+
+if (!window.Wasm) {
+  document.write('No WASM support. Did you enable it via chrome://flags/#enable-webassembly ?')
```

Comment:
please link to https://github.com/kripken/emscripten/wiki/WebAssembly#testing-native-webassembly-in-browsers which has those chrome instructions plus others.